src/test/java/MyBaseTest.java

```diff
@@ -0,0 +1,118 @@
+        submitButton.click();
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
+        passwordField.clear();
+        passwordField.sendKeys(password);
+    }
+
+    public static void provideEmail(String email) {
+        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
+        emailField.clear();
+        emailField.sendKeys(email);
+    }
+
+    public static void clickSaveButton() {
+        WebElement saveButton = driver.findElement(By.cssSelector("button.btn-submit"));
+        saveButton.click();
+    }
+
+    public static void provideProfileName(String randomName) {
+        WebElement profileName = driver.findElement(By.cssSelector("[name='name']"));
+        profileName.clear();
+        profileName.sendKeys(randomName);
+    }
+
+    public static void provideCurrentPassword(String password) {
+        WebElement currentPassword = driver.findElement(By.cssSelector("[name='current_password']"));
+        currentPassword.clear();
+        currentPassword.sendKeys(password);
+    }
+
+    public static String generateRandomName() {
+        return UUID.randomUUID().toString().replace("-", "");//
+    }
+
+    public static void clickAvatarIcon() {
+        WebElement avatarIcon = driver.findElement(By.cssSelector("img.avatar"));
+        avatarIcon.click();
+
+    }
+
+    public void searchSong(String songTitleKeyword) throws InterruptedException {
+        WebElement searchField = driver.findElement(By.cssSelector("div#searchForm input[type='search']"));
+        searchField.sendKeys(songTitleKeyword);
+        Thread.sleep(2000);
+    }
+
+    public void viewAllResults() throws InterruptedException {
+        WebElement viewAllButton = driver.findElement(By.cssSelector("div.results section.songs h1 button"));
+        viewAllButton.click();
+        Thread.sleep(3000);
+    }
+    public void selectFirstSong() throws InterruptedException {
+        WebElement viewAllFirstSong = driver.findElement(By.cssSelector("#songResultsWrapper div.item-container table tr:nth-child(2) td.title"));
+        viewAllFirstSong.click();
+        Thread.sleep(2000);
+    }
+
+    public void clickAddToButton() throws InterruptedException {
+        WebElement addToButton = driver.findElement(By.cssSelector("button.btn-add-to"));
+        addToButton.click();
+        Thread.sleep(2000);
+    }
+    public void choosePlaylist(String playlistName) throws InterruptedException {
+        WebElement playlistNameElement = driver.findElement(By.xpath("//section[@id='songResultsWrapper']//section/ul/li[contains(text(),'"+playlistName+"')]"));
+        playlistNameElement.click();
+        Thread.sleep(2000);
+    }
+    public String getNotificationPopUp(){
+        WebElement notificationText = driver.findElement(By.cssSelector("div.success.show"));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210958864-bd12a0e1-75fb-4326-ac3d-b4a0c01a04c0.png)

![image](https://user-images.githubusercontent.com/104761482/210958939-ae2d9be1-ce12-41f1-8a53-968417ea8e5f.png)